src/lib/Nav.svelte

```diff
@@ -10,6 +14,13 @@
 		>
 	</a>
 
+	<input type="checkbox" id="toggle-menu" />
+	<label class="menu-icon" for="toggle-menu">
+		<a id="menu-icon">
+			<IconBars />
+		</a>
+	</label>
+
```

Comment:
Could you explain what this do?

Reply:
it uses a checkbox instead of javascript onclick to trigger the opening of the menu, and thus works completely without js

Reply:
:+1: